css/stylesheet.css

```diff
@@ -47,6 +47,15 @@ header {
   z-index: 1;
 }
 
+.menuIcon{
+  /*display: none;*/
+  height: 40px;
+  width: 40px;
+  float: left;
+  margin: 20px;
+  z-index: 999;
```

Comment:
I owe you an answer to a better solution to this problem. I'm positive an `<img>` tag in the HTML isn't the answer. I promise I'll get to this and hopefully have something for you next week. This should solve the issue of the menu icon being behind the image.